build/dependencies.props

```diff
@@ -6,6 +6,18 @@
     <MicrosoftNETCoreDotNetAppHostPackageVersion>2.1.7</MicrosoftNETCoreDotNetAppHostPackageVersion>
  </PropertyGroup>
 
+ <PropertyGroup Label="Package Versions: Pinned">
+    <!--
+      The Razor Design \ Extensions package provide build assets (targets and tasks). A "Reference" \ "ProjectReference" does not
+      flow build assets, making it difficult to convert existing PackageReferences. These properties are listed to allow projects to reference these as packages.
+
+      Note that this section appears before DotNetPackageVersionPropsPath import. This allows GeneratedPackageVersionPropsPath to override it with current
+      version values when available.
+     -->
+    <MicrosoftAspNetCoreRazorDesignPackageVersion>2.1.1</MicrosoftAspNetCoreRazorDesignPackageVersion>
```

Comment:
shouldn't this be 2.1.2 now?

Reply:
Talked to Pranav offline about this. This should be just some fixed version, so 2.1.1 is fine.